api/internal/core/store/storehub.go

```diff
@@ -144,5 +145,17 @@ func InitStores() error {
 		return err
 	}
 
+	err = InitStore(HubKeyServerInfo, GenericStoreOption{
+		BasePath: "/apisix/data_plane/server_info",
+		ObjType:  reflect.TypeOf(entity.ServerInfo{}),
+		KeyFunc: func(obj interface{}) string {
```

Comment:
Style: should keep the indent same when initializing the structure.

Reply:
I have tried to fix it, but it will be recovery by `goland`.
So I think it's the behavior of `goland`.